packages/admin-ui/src/combobox/selected-multiple.state.ts

```diff
@@ -0,0 +1,50 @@
+import type { Dispatch, SetStateAction } from 'react'
+import { useState } from 'react'
+
+export function useCheckboxObjectState<T>(
```

Comment:
Maybe `useMultipleSelectionState` ? You don't necessarily need to render a checkbox.